core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestrator.scala

```diff
@@ -57,15 +61,22 @@ class BillingProjectOrchestrator(ctx: RawlsRequestContext,
    * Creates a "v2" billing project, using either Azure managed app coordinates or a Google Billing Account
    */
   def createBillingProjectV2(createProjectRequest: CreateRawlsV2BillingProjectFullRequest): Future[Unit] = {
+    def tagAndCaptureSentryEvent(e: Throwable): Unit = {
+      val sentryEvent = new SentryEvent()
+      sentryEvent.setTag("team", "workspaces")
```

Comment:
Small bit of pushback on the nomenclature here; I don't think we should bake the notion of the specific team into this code but rather the functional area of concern. That is, there may come a day when the "Broad Workspaces" team is no longer responsible for billing project failures and it'd be a bit of a pain to have to ship a PR to reflect that reality. 


Perhaps ("component","billing")?  Swat tests are setup [similarly](https://github.com/broadinstitute/rawls/blob/develop/automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsTestGroupAnnotations.java).